README.md

```diff
@@ -25,7 +25,7 @@ follows.
 Add the following to your `~/.bashrc`, adjusting any paths as necessary.
 
 ```
-export IDA_ANSIBLE_DIR="${HOME}/Projects/ida-ansible"
```

Comment:
I'm wondering if we should keep the environment variable the same given people will have it in their bash/zshrc files already...

Reply:
Is it worth supporting both, and only documenting `VERIFY_ANSIBLE_DIR`? If that's too much faff I don't think it's too bad to make the breaking change and force everyone to update their rc files.

Reply:
I'm not really bothered either way.  I personally feel that there are, at most, a few dozen people affected by this change and that it starts eliminating some of the cognitive overhead of having a blend of ida and verify everywhere. 

But I defer to a permanent member of staff on this one.  My tenure is likely to be shorter than the lifetime of this tooling.

Reply:
And it will tell you that it now needs to be `VERIFY_ANSIBLE_DIR`. I say LGTM. @richardTowers?